core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/WhineyFunction.java

```diff
@@ -24,7 +24,7 @@
         return function.apply(arg);
       } catch (IOException e) {
         logger.error("Ignoring error", e);
-        return null;
+        throw new RuntimeException(e);
```

Comment:
Now says "Ignoring error," then throws a RuntimeException (not ignoring the error), and will end up logging the stack trace twice